content/cloud-docs/api-docs/variable-sets.mdx

```diff
@@ -128,12 +128,24 @@ curl \
 
 ## Update a Variable Set
 
-`PUT/PATCH varsets/:varset_id`
+`PATCH varsets/:varset_id`
 
 | Parameter    | Description         |
 | ------------ | ------------------- |
 | `:varset_id` | The variable set ID |
 
+### Request Body
+
+Properties without a default value are required.
+
+| Key path                        | Type    | Default | Description                                                                                                                             |
+| ------------------------------- | ------- | ------- | --------------------------------------------------------------------------------------------------------------------------------------- |
+| `data.name`                     | string  |         | The name of the variable set.                                                                                                           |
+| `data.description`              | string  | `""`    | Text displayed in the UI to contextualize the variable set and its purpose.                                                             |
+| `data.global`                   | boolean | `false` | When true, Terraform Cloud automatically applies the variable set to all current and future workspaces in the organization.             |
+| `data.relationships.workspaces` | array   | \[]     | Array of references to workspaces that the variable set should be assigned to. Sending an empty array clears all workspace assignments. |
+| `data.relationships.vars`       | array   | \[]     | Array of complete variable definitions that comprise the variable set.                                                                  |
```

Comment:
- The paths are wrong for several of these properties -- the ones that aren't relationships should be `data.attributes.thing`, not `data.thing`, as per the structure of the example payload. This error affects some other tables on this page as well.
- Since this is a patch request, the `default` value for all of these properties should be `(previous value)` -- compare to the tables in the workspaces API page, for example. 
- "Array of complete variable definitions" is a very strange way to word this, and is likely confusing to readers. (I know that wording came from the table for the create endpoint; it's probably a good idea to change both.) What I can gather from going back up to the "create" endpoint is, this relationships array doesn't work like most of the relationships in the API and instead requires full filled-out `vars` objects, but we're not pointing to a useful definition of those objects. I think there's two possible approaches for making this nice: 
    - Add a section to this page that describes the vars objects, and refer to it from the various endpoints that interact with it. (This almost already exists, in the content for the add/update a variable endpoints further down the page.)
    - Add the sub-properties of the vars objects to these tables, using the `data.relationships.vars[].key` etc. notation that we've used in some other places to list the properties of objects in an array. (See for example [the payload for workspace remote state consumers](https://www.terraform.io/cloud-docs/api-docs/workspaces#request-body-7).)